### PR TITLE
Changes for tralis

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: src/components/RouteSchedule/RouteSchedule.js [hunk 1]
```diff
@@ -248,15 +251,20 @@ const renderStations = (lineInfos, onStationClick, trackerLayer) => (
...
   className,
   lineInfos,
-  header,
-  stations,
+  renderHeader,
+  renderStations,
   onStationClick,
   trackerLayer,
 }) {
+  document.documentElement.style.setProperty('--mask-height', `25px`);
```
Comment (edellucien) [inline src/components/RouteSchedule/RouteSchedule.js]:
```suggestion
```

File: src/layers/Tracker.js [hunk 2]
```diff
@@ -234,7 +178,21 @@ export default class Tracker {
           // We set the rotation and the timeFraction of the trajectory (used by tralis).
           this.trajectories[i].rotation = rotation;
           this.trajectories[i].endFraction = timeFrac;
+
+          // It happens than the now date was some ms before the first timeIntervals we have.
```
Comment (edellucien) [inline src/layers/Tracker.js]:
```suggestion
          // It happens that the now date was some ms before the first timeIntervals we have.
```

File: src/layers/Tracker.js [hunk 3]
```diff
@@ -259,6 +217,8 @@ export default class Tracker {
           hoverVehicleImg = vehicleImg;
           hoverVehiclePx = px;
         }
+      } else {
+        console.log('pas coord', coord);
```
Comment (edellucien) [inline src/layers/Tracker.js]: Maybe temporary log to remove?

Comment (oterral) [reply src/layers/Tracker.js]: Fixed

File: src/components/RouteSchedule/RouteSchedule.js (hunk 1 above)
Comment (oterral) [reply src/components/RouteSchedule/RouteSchedule.js]: Fixed

**Pull request merged**